activiti-cloud-services-runtime-bundle/activiti-cloud-services-subscriptions/src/main/java/org/activiti/services/subscriptions/behavior/BroadcastSignalEventActivityBehavior.java

```diff
@@ -29,6 +29,9 @@ public BroadcastSignalEventActivityBehavior(ApplicationEventPublisher eventPubli
 
     public void execute(DelegateExecution execution) {
         super.execute(execution);
+        if (processInstanceScope) {
```

Comment:
@daisuke-yoshimoto We should fix this behavior to trigger signal internally only for process scoped signals. Otherwise, we broadcast the signal via cloud stream that triggers signal via stream listener.

Reply:
@daisuke-yoshimoto 👍